src/simulators.jl

```diff
@@ -320,3 +321,106 @@ function simulate!(sys,
     end
     return sys
 end
+
+"""LangevinSplitting(; <keyword arguments>)
+A Langevin simulator using a general splitting scheme, consisting of a succession of **A**, **B** and **O** steps, corresponding respectively to updates in position, velocity for the potential part, and velocity for the thermal fluctuation-dissipation part. The `Langevin` and `VelocityVerlet` simulators without coupling correspond to the **BAOA** and **BAB** schemes respectively.
+# Arguments
+- `dt::dtType`: The timestep for the simulation
+- `friction::frictionType`: The friction coefficient. If units are used, it should have a dimensionality of mass per time.
+- `temperature::temperatureType`: The equilibrium temperature.
+- `splitting::splittingType`: The splitting specifier. Should be a string consisting of the characters `A`,`B` and `O`. Strings with no `O`s reduce to deterministic symplectic schemes.
+- `remove_CM_motion::Bool=true`: Whether to remove the centre of mass motion at each simulation iteration.
+- `rseed::UInt32=round(UInt32,time())`: An optional seed for the random number generator. Defaults to nearest epoch time in seconds.
+- `rng::rngType=MersenneTwister`: The random generator type for the simulation. The RNG is seeded with rseed.
+"""
+struct LangevinSplitting{dtType,frictionType,temperatureType,rngType,splittingType}
+    dt::dtType
+    friction::frictionType
+    temperature::temperatureType
+    rng::rngType
```

Comment:
I think the Langevin variants should be consistent in how they use the random seed. Passing it to `simulate!` as a kwarg might be more flexible than having it in the struct, since a user may make repeated calls to `simulate!` with the same simulator and wish to control the random seed each time.